plugins/com.google.cloud.tools.eclipse.dataflow.ui/src/com/google/cloud/tools/eclipse/dataflow/ui/preferences/RunOptionsDefaultsComponent.java

```diff
@@ -432,6 +432,11 @@ public void selectionChanged(SelectionChangedEvent event) {
     stagingLocationInput.addModifyListener(listener);
   }
 
+  public boolean isEnabled() {
```

Comment:
This is used only in tests. Probably don't have to be `public` but `@VisibleForTesting`.

Reply:
Still pending.

Reply:
Weird. I can't compile unless this is `public`, but Travis doesn't seem to have any problem. Should we revert in to `public`? `@VisibleForTesting` could remain.